imports/api/logic/grad-checker/faculty/computing/computer-science/AY-1516/AY1516GradCheckerHandler.js

```diff
@@ -86,11 +86,12 @@ export const AY1516CSGradChecker = function AY1516CSGradChecker(studentSemesters
   const allGraduationRequirementMCs = getGradRequirementMCs(cohortGradRequirementIDs);
 
   // retrieve focus area modules
-  const allFocusAreaPrimaryRequirements = getFocusAreaPrimaryRequirement(cohortInformation.cohortFocusAreaID);
-  const allFocusArea4KRequirements = getFocusArea4KRequirement(cohortInformation.cohortFocusAreaID);
-  const allFocusArea4KPrimaryRequirements = getFocusAreaPrimary4KRequirement(cohortInformation.cohortFocusAreaID);
-  const allFocusAreaNonPrimaryRequirements = getFocusAreaNonPrimaryRequirement(cohortInformation.cohortFocusAreaID);
-
+  const allStudentFocusAreas = {
```

Comment:
Looks good! More complicated attributes are better in a JSON object than a list of variable declarations!